Resources/Prototypes/Entities/Mobs/Player/humanoid.yml

```diff
@@ -277,6 +293,8 @@
   id: ERTEngineer
   parent: ERTLeader
   components:
+    - type: RandomHumanoidAppearance
+      randomizeName: false
     - type: MindShield
     - type: GhostRole
       name: ghost-role-information-ert-engineer-name
```

Comment:
Just make a new parent and use that instead of copy-pasting the component 20 times please. Same deal with initial infected exempt + trakeoveravailable + mindshield.

Reply:
Small problem, randomHumanoidSettings doesn't seem to support inheritance? I did try doing that but it didn't seem to work.

Reply:
Nvm, I fixed it somehow.